mlir/utils/performance/tuningRunner.py

```diff
@@ -181,6 +181,18 @@ def extractFusionConfigs(test_dir, paths: Paths, options: Options):
 
     return opType
 
+def getExtendedGemmConfigurations(filename):
```

Comment:
I don't think I like the idea of implicitly adding more configs to tune

Would it make more sense to explicitly add the `i8 * i8 -> i8` configs to our files of things we want to tune?

Reply:
THe problem here is that we are extending our tuning time by 30%. 

The way I see it is that you are adding another implicit datatype. In other words we have: f32, f16, i8i8, i8i32. Can you parametrize which datatype we want to tune? In my recently merged PR (https://github.com/ROCmSoftwarePlatform/rocMLIR/pull/1100) I have introduced a `--datatype` parameter to the tuning runner. Maybe you can remove `i8` and add `i8i8` and `i8i32` . The deafult might be all of them (or maybe just `i8i8`)

Reply:
Yes, because that's what we need.